helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -37,11 +37,22 @@ spec:
           command:
             - /manager
           args:
-          {{- if .Values.batchScheduler.enabled }}
-            - --enable-batch-scheduler
-          {{- else }}
-            []
-          {{- end }}
+            {{- $argList := list -}}
+            {{- if .Values.batchScheduler.enabled -}}
+            {{- $argList = append $argList "--enable-batch-scheduler" -}}
+            {{- end -}}
+            {{- $watchNamespace := "" -}}
+            {{- if .Values.singleNamespaceInstall -}}
+            {{- $watchNamespace = .Release.Namespace -}}
+            {{- else if .Values.watchNamespace -}}
```

Comment:
watchNamespace is for a slightly different (but related case) where Ray users should be scoped to a single namespace (that is different to the operator namespace).

I'd like to add this because otherwise that flag is inaccessible via helm.

Reply:
Got it. Thank you!